homework/index.js

```diff
@@ -28,26 +30,133 @@
     });
     return elem;
   }
+  function formatDate(dateString) {
+    const dateTime = new Date(dateString);
+    return dateTime.toLocaleString();
+  }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function renderError(error) {
+    const container = document.getElementById('main-container');
+    container.innerHTML = '';
+    createAndAppend('div', container, {
+      text: error.message,
+      class: 'alert',
+    });
   }
 
-  function main(url) {
-    fetchJSON(url, (err, repos) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, {
-          text: err.message,
-          class: 'alert-error',
+  function addTable(table, body, value) {
```

Comment:
You are not actually adding a table here, just a table row. So a better name would be `addRow` or `addTableRow`.